### PR TITLE
Added documentation to Login Logout

### PR DESCRIPTION
**Pull request by stevenshuhyow**

### CODE REVIEW

**Comment by kevhuang** [comment]
We'll need to wait for @stevenshuhyow to recommit some changes and also squash some commits.

File: server/server.js [hunk 1]
```diff
@@ -10,20 +10,31 @@ process.env.NODE_ENV = process.env.NODE_ENV || 'development';
...
 var parser = require('body-parser');
 var config = require('./config/environment');
+var session = require('express-session');
 var passport = require('passport');
 var db = require('./db');
+var cookieParser = require('cookie-parser');
+var passport = require('passport');
+
+
+db.init();
```
Comment (kevhuang) [inline server/server.js]: This is duplicated. Let's remove.

File: server/server.js [hunk 2]
```diff
@@ -33,5 +44,13 @@ server.listen(config.port, config.ip, function () {
   console.log('Express server listening on %d, in %s mode', config.port, app.get('env'));
 });
 
+
+// Init database
+
+
+app.use(passport.initialize());
+app.use(passport.session());
```
Comment (kevhuang) [inline server/server.js]: Based on how we set up the server and config files before, I think the middleware features for Express should be defined in config/express.js. As of right now, there are some things that are duplicated in this file and express.js too. You can also remove any duplicated `require()` statements from this file if they're going to be used in express.js.

File: server/auth/authController.js [hunk 3]
```diff
@@ -16,33 +16,35 @@ module.exports = function(passport){
   clientSecret: localServices.facebook.secret,
   callbackURL: client.facebookAuth.callbackURL
```
Comment (kevhuang) [inline server/auth/authController.js:17]: Can you let us know what callback URL we need to manually hard code for now when testing?

Never mind, I found it: http://localhost:3000/auth/facebook/callback

**Comment by dmsakamoto** [comment]
:+1:

**Pull request merged**